typescript/frontend-cicd/README.md

```diff
@@ -0,0 +1,52 @@
+
+# CI/CD Pipeline for SPA Frontend
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+> **This is a stable example. It should successfully build out of the box**
+>
+> This examples only on the core CDK library, and does not have any infrastructure prerequisites to build.
+
+---
+<!--END STABILITY BANNER-->
+
+![Frontend-Pipeline](https://d1zrwss8zuawdm.cloudfront.net/frontend-cicd.png)
+
+This example constructs above architecture with CDK.
+You can start simple frontend CI/CD pipeline for ReactJS with this CDK.
```

Comment:
What will the customer see? How will the customer know that they have done everything successfully?

Explain the process from cloning the repository to seeing the page hosted through CloudFront.

Reply:
I have added additional description as you requested. please check. :)